README.md

```diff
@@ -1,7 +1,7 @@
-# Run0
+# Kaps
 
 <img src="https://img.shields.io/github/workflow/status/virt-do/run0/run0%20build%20and%20unit%20tests?style=for-the-badge" />
```

Comment:
The badge link should be updated 😉 

Update it to : `https://img.shields.io/github/workflow/status/virt-do/kaps/run0%20build%20and%20unit%20tests?style=for-the-badge`

It ends with `run0` as I think the badge's name hasn't been updated in the repo for now, so the link is working this way. Waiting confirmation for that @sameo